src/pymatgen/analysis/local_env.py

```diff
@@ -4075,6 +4075,35 @@ def _semicircle_integral(dist_bins, idx):
 
         return (area1 - area2) / (0.25 * math.pi * radius**2)
 
+    @staticmethod
```

Comment:
can you add a link to your issue to the `_semicircle_integral` doc string for later reference?

also, would be good to type-hint both the old and your new method